ios/CustomerioInAppMessaging.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import React
+
+@objc(CustomerioInAppMessaging)
+class CustomerioInAppMessaging: RCTEventEmitter {
+    public static var shared: CustomerioInAppMessaging?
```

Comment:
```suggestion
    public static let shared: CustomerioInAppMessaging = CustomerioInAppMessaging()
```

This is common to see in Swift code for singletons - have it be immutable and set a value in-line. 

But, I am OK with the way it is currently if there is a reason.

Reply:
Thanks, this looks clean. I'll give it a try and see if everything works as expected.

Reply:
This doesn't seem to be working with react native. I think it is mainly because native modules registered with react native have to be initialized late. I have reverted the change and added inline comment to it.